lib/web/apiserver.go

```diff
@@ -742,6 +745,7 @@ func (h *Handler) getWebConfig(w http.ResponseWriter, r *http.Request, p httprou
 		Providers:        authProviders,
 		SecondFactor:     secondFactor,
 		LocalAuthEnabled: clsCfg.GetLocalAuth(),
+		AuthType:         h.cfg.AuthType,
```

Comment:
Does `cap.GetType()` returns something else? Why do we need to pass AuthType if you can get it from the cap object above?

Reply:
you're right, revised